examples/rules_cc/WORKSPACE

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Uber Technologies, Inc.
+# Licensed under the MIT License
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+HERMETIC_CC_TOOLCHAIN_VERSION = "v2.0.0-rc1"
```

Comment:
We need a test which ensures this version is bumped when we cut new releases.

Reply:
Let's add this in #58